google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Subscriber.java

```diff
@@ -450,7 +449,6 @@ public void run() {
             .build();
 
     SubscriptionName subscriptionName;
-    Optional<Credentials> credentials = Optional.absent();
```

Comment:
Minor:
Not related to this specific change, but this removes an Optional<> field and there is one more Optional<> field in this class: Optional<ApiClock> clock. Using optionals for class fields is considered a bad practice (for example: http://stackoverflow.com/questions/23454952/uses-for-optional), plus Optional is not serializable and can cause problems for serializable classes. Can you also change clock's optional to a regular field as part of this PR?